src/main/resources/BankOfSpringAppData.xml

```diff
@@ -0,0 +1,121 @@
+						<!-- using property -->
+						<property name="maxBalanceAmount" value="170000000"></property>
+					</bean>
+					
+				</entry>
+				<entry key="account2">
+					<bean id="account2" class="org.bankofspring.model.Account">
+						<constructor-arg name="owningCustomers">
+							<list>
+								<ref bean="customer1" />
+							</list>
+						</constructor-arg>
+						<constructor-arg name="accountNumber" value="account2" />
+						<constructor-arg name="accountDescription" value="account2description" />
+						<property name="maxBalanceAmount" value="100000000"></property>
+					</bean>
+				</entry>
+
+			</map>
+		</property>
+	</bean>
+
+	<bean id="customer2" class="org.bankofspring.model.Customer">
+		<constructor-arg name="username" value="user2" />
+		<constructor-arg name="password" value="test" />
+		<!--  value as an element on its own -->
+		<constructor-arg name="customerID">
+			<value>customer2</value>
+		</constructor-arg>
+		<constructor-arg name="name" value="customertwo" />
+
+		<property name="accounts">
+			<map value-type="org.bankofspring.model.Account">
+			<!-- note use of SpEL to use accoutn number from account3 and 4 as key for the map -->
+				<entry key="#{account3.accountNumber}" value-ref="account3"></entry>
+				<entry key="#{account4.accountNumber}" value-ref="account4"></entry>
+			</map>
+		</property> 	
+
+	</bean>
+<!--  2 accounts for customer 2-->
+   <bean id="account3" class="org.bankofspring.model.Account">
+
+		<constructor-arg name="accountNumber" value="account3" />
+		<constructor-arg name="accountDescription" value="account3description" />
+		<property name="maxBalanceAmount" value="150000000"></property>
+		<property name="accountBalance" value="100"></property>
+		<constructor-arg name="owningCustomers">
+			<list>
+				<ref bean="customer2" />
+			</list>
+		</constructor-arg>
+   </bean>
+
+	<bean id="account4" class="org.bankofspring.model.Account">
+
+		<constructor-arg name="accountNumber" value="account4" />
+		<constructor-arg name="accountDescription" value="account4description" />
+		<property name="maxBalanceAmount" value="100000000"></property>
+		<property name="accountBalance" value="150"></property>
+		<constructor-arg name="owningCustomers">
+			<list>
+				<ref bean="customer2" />
+				<ref bean="customer1" />
+			</list>
+		</constructor-arg>
+		<property name="transactions">
+			<list>
+				<ref bean="txn1"></ref>
+			</list>
+		</property>
+
+	</bean>
+
+	<!-- A transaction for customer2 account 4 -->
+	<bean id="txn1" class="org.bankofspring.model.AccountTransaction">
+		<constructor-arg name="toAccount" ref="account4" />
+		<constructor-arg name="fromAccount">
+			<null />
+		</constructor-arg>
+		<constructor-arg name="transactionAmount" value="100"></constructor-arg>
+<!-- another example of SpEl. this time with java.util.Calendar -->
+		<property name="transactionDate"		
+			value="#{new java.util.GregorianCalendar(2015,1,1).getTime()}" />
+
+	</bean>
+	
+	
+
+</beans>
```

Comment:
Getting to the main point of the exercise, though, this is a pretty good config file. Constructor args, properties, maps, lists, nulls, and even some SpEL. Good stuff!